packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx

```diff
@@ -245,19 +246,16 @@ const ConditionsWidget: React.FC<ConditionsWidgetProps> = ({
                       placeholder={t('searchConditions', 'Search conditions')}
                       ref={searchInputRef}
                       renderIcon={errors?.conditionName && ((props) => <WarningFilled fill="red" {...props} />)}
-                      value={(() => {
-                        if (selectedCondition) {
-                          return selectedCondition.display;
-                        }
-                        if (debouncedSearchTerm) {
-                          return value;
-                        }
-                      })()}
+                      value={selectedCondition ? selectedCondition.display : debouncedSearchTerm ? value : ''}
```

Comment:
This change breaks the conditions search functionality:

https://github.com/user-attachments/assets/28d6bbde-51c7-4bfc-9498-5609c7455320

Reply:
Hi @denniskigen , I've restored the old version. Thanks!